---
layout: post
title: Chapter Published - Analysis Methods for Shotgun Metagenomics
author: Jake Price
---

Our book chapter titled "Analysis Methods for Shotgun Metagenomics", and its containing book (Theoretical and Applied Aspects of Systems Biology), is/are now available online! I'd like to thank Steve for leading the charge on this project as well as inviting me to contribute to it. 

PDFs ([chapter]({{ site.url }}/assets/publications/Woloszyneketal-2018.pdf) and the complete [book]({{ site.url }}/assets/publications/2018_Book_TheoreticalAndAppliedAspectsOf.pdf)) can be found on my [Publications](https://jacobrprice.github.io/2-publications.html) page or through the [Springer website](https://link.springer.com/chapter/10.1007/978-3-319-74974-7_5).   

[Analysis Methods for Shotgun Metagenomics](https://link.springer.com/chapter/10.1007/978-3-319-74974-7_5)   
Stephen Woloszynek, Zhengqiao Zhao, Gregory Ditzler, **Jacob R. Price**, Erin R. Reichenberger, Yemin Lan, Jian Chen, Joshua Earl, Saeed Keshani Langroodi, Garth Ehrlich, & Gail Rosen (2018) Analysis Methods for Shotgun Metagenomics. In: Alves Barbosa da Silva F., Carels N., Paes Silva Junior F. (eds) Theoretical and Applied Aspects of Systems Biology. Computational Biology, vol 27. Springer, Cham. doi:[10.1007/978-3-319-74974-7_5](https://doi.org/10.1007/978-3-319-74974-7_5)     

**Absract**   
The development of whole metagenome shotgun sequencing (WGS) has enabled the precise characterization of taxonomic diversity and functional capabilities of microbial communities in situ while obviating organism isolation and cultivation procedures. WGS created with second- and third-generation sequencing technologies will generate millions of reads and tens (or hundreds) of gigabytes of information about the organisms under investigation. Despite containing an immense amount of information, the reads are unorganized and unlabeled, leading to a significant challenge in discerning from which genome a read originated. Thus, analysis of WGS data necessitates first determining community structure and function from the raw reads before the focus can shift to making multi-sample comparisons. A typical WGS workflow consists of read assignment (taxonomic binning and classification), preprocessing techniques (normalization, dimensionality reduction), exploratory approaches (feature selection and extraction, ordination), statistical inference (regression, constrained ordination, differential abundance analysis), and machine learning. The following chapter provides an overview of these analytical approaches (including challenges and possible pitfalls that may be encountered by researchers) as well as steps toward their solutions. Relevant software packages and resources are also discussed.

